modules/client-configuration/pages/channels.adoc

```diff
@@ -0,0 +1,55 @@
+[[channels]]
+= Software Channels
+// Antora entities
+ifndef::backend-pdf[]
+include::ROOT:partial$entities.adoc[]
+endif::[]
+
+// PDF entities
+ifdef::backend-pdf[]
+include::_partials/entities.adoc[]
+endif::[]
+
+// Originally copied from reference-webui-channels.adoc. LKB 2019-03-11
+
+A software channel provides packages grouped by products or applications to simplify the selection of packages to be installed on a system.
+
+There are two types of software channels: base channels and child channels.
+
+Base Channels::
+A base channel consists of packages built for a specific architecture and release.
+For example, all of the packages in {sls}{nbsp}12 for the `x86_64` architecture make up a base channel.
+The list of packages in {sls}{nbsp}12 for the `s390x` architecture make up a different base channel.
++
+
+A system must be subscribed to only one base channel assigned automatically during registration based on the {sle} release and system architecture.
+For paid base channels, an associated subscription must exist.
+
+Child Channels::
+A child channel is associated with a base channel and provides extra packages.
+For example, an organization can create a child channel associated with {sls} on `x86_64` architecture that contains extra packages for a custom application.
```

Comment:
This statement sounds more like the difference between Vendor and Custom Channels.

Please remember:

At SUSE we have always a `Pool` and an `Update` channel. The Pool defines in most cases the base channel while the Update channel is already a child. And with SLE15 the situation is more scary. The Base Channel just contains 3 packages while the `SLE-Module-Basesystem` (Pool and Update - both childs) contains the important parts of the packages.